src/workloads/query/LookupColocatedData.yml

```diff
@@ -144,14 +144,12 @@ Actors:
           OperationCommand:
             shardCollection: test.ShardedLocal
             key: {*LocalShardKey: hashed}
-            numInitialChunks: 6
         # Here we use 1 chunk. This ensures that all the data for this coll is on one shard.
         - OperationMetricsName: ShardLocalCollectionAllOnOneShard
           OperationName: AdminCommand
           OperationCommand:
             shardCollection: test.ShardedLocalAllOnOneShard
-            key: {*LocalShardKey: hashed}
-            numInitialChunks: 1
+            key: {*LocalShardKey: 1}
```

Comment:
Pointing out for other reviewers the chose around this shard key change: since `numInitialChunks` was being ignored, the chunks were actually created on all shards so the distributed lookup was faster (because it was not happening on just one node) and performances improved.
Changing the shard key to non-hashed actually forces the collection to be created on one node and brings performances back to "normal"

Reply:
Thanks! I was wondering why the numInitialChunks params were being deleted.